flink-python/pyflink/common/types.py

```diff
@@ -177,7 +177,10 @@ def of_kind(row_kind: RowKind, *args, **kwargs):
         return row
 
     def __contains__(self, item):
-        return item in self._values
+        if hasattr(self, "_fields"):
+            return item in self._fields
```

Comment:
The enhancement is great. I have one comment, the `_fields` could be a None by invoking `set_field_names`. So it should return `False` in this case.

Reply:
so then the question is, if `_fields` is None do we want to fallback and do a membership check for the values or always just return False?

Reply:
@wzorgdrager Thanks a lot for the PR. Personally I prefer to fallback to check for the values if `_fields` is None.

Reply:
@dianfu I agree. I added the change now.